wazo_dird/plugins/phonebook_backend/http.py

```diff
@@ -56,15 +104,33 @@ def delete(self, source_uuid):
     def get(self, source_uuid):
         return super().get(source_uuid)
 
+    def _prepare_source_update(self, source_data: dict) -> PhonebookSourceInfo:
+        assert 'phonebook_uuid' in source_data
```

Comment:
the assertion should be removed to avoid having to handle a key error and an assertion error depending on the execution context

Reply:
https://github.com/wazo-platform/wazo-dird/pull/181#discussion_r1340229950
https://mm.wazo.community/wazo/pl/83pgontwpfgqdg5wod17e4c1xo

Reply:
@DrPyser so is it resolved ?